UI Session 8/doublyLinkedList.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <script>
+        function Node(data) {
+            this.data = data;
+            this.next = null;
+            this.pre = null;
+        }
+
+        function LinkedList() {
+            this.head = null;
+        }
+
+        LinkedList.prototype.add = function (data) {
+
+            var newNode = new Node(data);
+            var previousNode = null;
+            if (this.head == null) {
+                this.head = newNode;
+            }
+            else {
+                var current = this.head;
+                while (current.next != null) {
+                    previousNode = current;
+                    current = current.next;
+                }
+                current.next = newNode;
+                current.pre = previousNode;
+            }
+        }
+
+        LinkedList.prototype.delete = function () {
+            var current = this.head;
+            var previousNode = null;
+            while (current.next != null) {
+                previousNode = current;
+                current = current.next;
+            }
+
+            if (current == this.head) {
+                this.head = null;
+            } else {
+                current.pre = null;
+                previousNode.next = null;
+            }
+        }
+
+
+        LinkedList.prototype.printList = function () {
+            var current = this.head;
+            var string = "";
+            while (current != null) {
+                string += current.data + " ";
+                current = current.next;
+            }
+            console.log(string);
+        }
+
+
+    </script>
+</head>
+
+<body>
+
```

Comment:
No html tag, if need to commit only script then why not use the JS file instead of html